css/app/datepicker.css

```diff
@@ -89,27 +89,35 @@
 	border-left: 0;
 }
 
-#datepicker table tbody tr.uib-weeks td:last-child button {
-	border-right: none;
+.today-hilight {
```

Comment:
Just one thing: Can you use the class exactly as I said in my previous comment:

> .highlight-today

(Proper english and also the correct order, as we use .highlight-weekend too)